web/components/nav/FcDashboardNav.vue

```diff
@@ -1,37 +1,37 @@
 <template>
-  <v-list
-    class="d-flex fill-height flex-column justify-center"
-    dense>
-    <FcDashboardNavItem
-      :active-route-names="[
-        'viewCollisionReportsAtLocation',
-        'viewDataAtLocation',
-        'viewStudyReportsAtLocation',
-      ]"
-      icon="map"
-      label="View Map"
-      :to="toViewMap" />
-    <FcDashboardNavItem
-      :active-route-names="[
-        'requestStudyBulkEdit',
-        'requestStudyEdit',
-        'requestStudyNew',
-      ]"
-      icon="clipboard-list"
-      label="Track Requests"
-      :to="{ name: 'requestsTrack' }" />
-    <FcDashboardNavItem
-      :badge="jobsExistsNew"
-      icon="download"
-      label="Manage Downloads"
-      :to="{ name: 'downloadsManage' }" />
+  <nav class="d-flex fill-height flex-column justify-center">
```

Comment:
drop this nav element, please - we already have one from the top-level `<v-navigation-drawer>`.